tests/testvectors.cpp

```diff
@@ -48,8 +48,8 @@ void check_testcase(const testcase_t &testcase) {
     if (tc.valid) {
         ASSERT_EQ(err, PARSER_OK) << parser_getErrorDescription(err);
     } else {
-        ASSERT_NE(err, PARSER_OK) << parser_getErrorDescription(err);
-        return;
+        if (err != PARSER_OK)
```

Comment:
why did we need to change this ?

Reply:
This was an obvious bug. If there is an error in the transaction, it can be detected either by .parse, or by .validate. The previous code required .parse to find the error, and the else branch of the .validate was unreachable.